app/views/catalogs/_logs.html.erb

```diff
@@ -75,3 +33,22 @@
     		$.SocketURL = "<%= Ind.websocket_tap %>";
      })($);
 	</script>
+
+<script src="https://cdn.socket.io/socket.io-1.2.0.js"></script>
```

Comment:
Is this the latest socket.io version  ? or move it up to the latest ? 
Can we not just use a `gem` as its called from the `html.erb` ?